README.md

```diff
@@ -5,7 +5,9 @@
 
 # Export Gerrit change request information
 
-The gerrit-change-info action takes the Gerrit change URL and patchset number an input and returns information about Gerrit change request that is reused as input in other releng reusable workflows.
+The gerrit-change-info action takes the Gerrit change URL and patchset number
+as input and returns information about the Gerrit change request, which other
+releng reusable workflows then reuse as input.
 
 
```

Comment:
Please remove line 12. Pre-commit is trying to do it in a follow-on to the PR but it fails because the bot does not sign it's commits